ch.threema.threema-desktop.yml

```diff
@@ -0,0 +1,72 @@
+app-id: ch.threema.threema-desktop
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '21.08'
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node14
+command: run.sh
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=network
+build-options:
+  append-path: /usr/lib/sdk/node14/bin
+  cflags: -O2 -g
+  cxxflags: -O2 -g
```

Comment:
```suggestion
```

Reply:
I set these as they were set in the sample electron flatpak yml.  If this is considered wrong, shall I also open a PR against the sample repo?

Reply:
I will just leave this as it is since that's what's recommended here:

https://docs.flatpak.org/en/latest/electron.html

Reply:
If @barthalion tells you to remove it, then you should. The example is out of date.
Also I don't see anything actually compiled.